source/MaterialXTest/Runtime.cpp

```diff
@@ -1180,6 +1180,14 @@ TEST_CASE("Runtime: Looks", "[runtime]")
     assign2.getExclusive().getValue().asBool() = false;
     look1.addMaterialAssign(pa);
     look1.addMaterialAssign(pa2);
+    try
+    {
+        look1.addMaterialAssign(col1.getPrim());
```

Comment:
minor: I should you have a REQUIRE(false); below this line just to make sure the function isn't returning successfully?